DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -99,5 +114,98 @@ EditLearningPathwayDefaultsViewModel model
 
             return RedirectToAction("Index", new { customisationId = model.CustomisationId });
         }
+
+        [HttpGet]
+        [ServiceFilter(typeof(RedirectEmptySessionData<EditLearningPathwayDefaultsData>))]
+        [Route("AutoRefreshOptions")]
+        public IActionResult EditAutoRefreshOptions(int customisationId)
+        {
+            var centreId = User.GetCentreId();
+            var categoryId = User.GetAdminCategoryId()!;
+
+            var courseDetails = courseService.GetCourseDetailsForAdminCategoryId(
+                customisationId,
+                centreId,
+                categoryId.Value
+            );
+
+            var model = new EditAutoRefreshOptionsViewModel(courseDetails!);
+
+            SetViewBagCourseOptions(customisationId, model.RefreshToCustomisationId);

```

Comment:
We shouldn't need to use ViewBag, these course options should go on the ViewModel, and then the data that is posted back to the endpoint will be a FormData model. 
See https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19806453895/View+Models+and+Form+Data

Reply:
Moved from View bag to the view model